lib/views/login.dart

```diff
@@ -0,0 +1,147 @@
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(15.0, 0, 15, 5),
+                      child: TextField(
+                        autofocus: true,
+                        cursorHeight: 20,
+                        style: Theme.of(context).textTheme.titleSmall,
+                        controller: email,
+                        decoration: const InputDecoration(
+                          border: OutlineInputBorder(),
+                          hintText: 'Deine E-Mail',
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(15.0, 0, 15, 15),
+                      child: TextField(
+                        obscureText: true,
+                        enableSuggestions: false,
+                        autocorrect: false,
+                        autofocus: false,
+                        cursorHeight: 5,
+                        style: Theme.of(context).textTheme.titleSmall,
+                        controller: passwort,
+                        decoration: const InputDecoration(
+                          border: OutlineInputBorder(),
+                          hintText: 'Deine Passwort',
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(0.0, 0.0, 0.0, 20.0),
+                      child: Text(
+                        "Du hast schon einen Account?",
+                        style: Theme.of(context).textTheme.titleSmall,
+                      ),
+                    ),
+                    Row(
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(right: 8.0),
+                          child: MaterialButton(
+                            shape: RoundedRectangleBorder(
+                                borderRadius: BorderRadius.circular(8)),
+                            height: MediaQuery.of(context).size.height * 0.05,
+                            minWidth: MediaQuery.of(context).size.height * 0.15,
+                            color: Theme.of(context).highlightColor,
+                            textColor: Colors.black,
+                            onPressed: () => {},
+                            splashColor: Colors.redAccent,
+                            child: Text(
+                              "Passwort vergessen",
+                              style: Theme.of(context).textTheme.titleSmall,
+                            ),
+                          ),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.only(left: 8.0),
+                          child: MaterialButton(
```

Comment:
Vielleicht in ferner Zukunft den Material Button disablen, wenn nicht alle Felder ausgefüllt sind, und eventuell email auf richtigkeit prüfen. Aber nur nth, evtl auch erst nächstes Semester